incubator/jaeger/templates/spark-cronjob.yaml

```diff
@@ -29,40 +29,42 @@ spec:
-            image: {{ .Values.spark.image }}:{{ .Values.spark.tag }}
-            imagePullPolicy: {{ .Values.spark.pullPolicy }}
-            env:
-            - name: STORAGE
-              value: {{ .Values.storage.type }}
-            {{- if eq .Values.storage.type "cassandra" }}
-            - name: CASSANDRA_CONTACT_POINTS
-              value: {{ template "cassandra.contact_points" . }}
-            - name: CASSANDRA_KEYSPACE
-              value: {{ printf "%s_%s" "jaeger_v1" .Values.cassandra.config.dc_name | quote }}
-              {{- end }}
-            {{- if eq .Values.storage.type "elasticsearch" }}
-            - name: ES_NODES
-              value: {{ template "elasticsearch.client.url" . }}
-            - name: ES_NODES_WAN_ONLY
-              value: {{ .Values.storage.elasticsearch.nodesWanOnly | quote }}
-            - name: ES_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: {{ if .Values.storage.elasticsearch.existingSecret }}{{ .Values.storage.elasticsearch.existingSecret }}{{- else }}{{ include "jaeger.fullname" . }}-elasticsearch{{- end }}
-                  key: password
-            - name: ES_USERNAME
-              value: {{ .Values.storage.elasticsearch.user }}
-            {{- end }}
-            resources:
-{{ toYaml .Values.spark.resources | indent 14 }}
+            - name: {{ include "jaeger.fullname" . }}-spark
+              securityContext:
+                  {{- toYaml .Values.spark.securityContext | nindent 16 }}
+              image: {{ .Values.spark.image }}:{{ .Values.spark.tag }}
+              imagePullPolicy: {{ .Values.spark.pullPolicy }}
+              env:
+                - name: STORAGE
+                  value: {{ .Values.storage.type }}
+                {{- if eq .Values.storage.type "cassandra" }}
+                - name: CASSANDRA_CONTACT_POINTS
+                  value: {{ template "cassandra.contact_points" . }}
+                - name: CASSANDRA_KEYSPACE
+                  value: {{ printf "%s_%s" "jaeger_v1" .Values.cassandra.config.dc_name | quote }}
+                  {{- end }}
+                {{- if eq .Values.storage.type "elasticsearch" }}
+                - name: ES_NODES
+                  value: {{ template "elasticsearch.client.url" . }}
+                - name: ES_NODES_WAN_ONLY
+                  value: {{ .Values.storage.elasticsearch.nodesWanOnly | quote }}
+                - name: ES_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ if .Values.storage.elasticsearch.existingSecret }}{{ .Values.storage.elasticsearch.existingSecret }}{{- else }}{{ include "jaeger.fullname" . }}-elasticsearch{{- end }}
+                      key: password
+                - name: ES_USERNAME
+                  value: {{ .Values.storage.elasticsearch.user }}
+                {{- end }}
+              resources:
+                {{ toYaml .Values.spark.resources | indent 16 }}
```

Comment:
Why is everything here indented? This is unrelated change, please, revert.

Reply:
My intention is that charts be somewhat standardized. The helm create command uses an indentation for lists. I actually preferred no indentation and made a PR https://github.com/helm/helm/pull/5725 ... However after getting feedback from at least 2 helm maintainers and some of my colleagues, the general consensus is that indented lists are more readable.

Reply:
Yes, I agree 100% that indentation is easier to read. However, it makes this particular PR harder to reader/review. I'd still prefer a separate PR with just the indentation and then rebasing this branch to reflect the changes.